tsconfig.json

```diff
@@ -16,9 +16,11 @@
     "target": "ES6",
   },
   "files": [
-    "src/index.ts"
+    "src/index.ts",
+    "public/index.ts"
   ],
   "include": [
-    "src/**/*"
+    "src/**/*",
+    "public/**/*"
```

Comment:
Question: What's the reason for having both of these? Is it not sufficient to just have the `include`?

Reply:
I honestly don't know the answer without reading the typescript compiler docs